src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -1,21 +1,23 @@
 import React, { Component, ReactNode, ReactElement } from 'react';
+import { default as styled } from 'styled-components';
 import classnames from 'classnames';
-import { HtmlLabel, HtmlLabelProps, HtmlSpan } from '../../reset';
-import { Paragraph, ParagraphProps } from '../Paragraph/Paragraph';
 import {
   ListboxInput,
   ListboxButton,
   ListboxButtonProps,
   ListboxPopoverProps,
   ListboxList,
-  ListboxOption,
   ListboxPopover,
 } from '@reach/listbox';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { baseStyles, listboxPopoverStyles } from './Dropdown.baseStyles';
+import { DropdownItemProps } from '../DropdownItem/DropdownItem';
+import { HtmlSpan } from '../../../reset';
+import { LabelText, LabelMode } from '../../Form/LabelText/LabelText';
+import { logger } from '../../../utils/logger';
+import { AutoId } from '../../../utils/AutoId';
 import { positionMatchWidth } from '@reach/popover';
```

Comment:
Imports should be ordered so that external imports are introduced first, then e.g. utils / resets, other components and then finally imports related directly to this component.

Reply:
Should now be in a more logical order.